Marlin/Marlin_main.cpp

```diff
@@ -1770,6 +1778,26 @@ static void homeaxis(AxisEnum axis) {
       }
     #endif
 
+    #ifdef BABYSTEPPING
+      if(axis == Z_AXIS)
+      {
+        baby_max_endstop[axis] = Z_BABY_DEFAULT_MAX_POS;
+        baby_min_endstop[axis] = Z_BABY_DEFAULT_MIN_POS;
+      }
+      #ifdef BABYSTEP_XY
+        else if(axis == X_AXIS)
+        {
+          baby_max_endstop[axis] = X_BABY_DEFAULT_MAX_POS;
+          baby_min_endstop[axis] = X_BABY_DEFAULT_MIN_POS;
+        }
+        else if(axis == Y_AXIS)
+        {
+          baby_max_endstop[axis] = Y_BABY_DEFAULT_MAX_POS;
+          baby_min_endstop[axis] = Y_BABY_DEFAULT_MIN_POS;
+        }
+      #endif //BABYSTEP_XY
+    #endif //BABYSTEPPING
```

Comment:
Formatting should be like so:
```
    #ifdef BABYSTEPPING
      if (axis == Z_AXIS) {
        baby_max_endstop[axis] = Z_BABY_DEFAULT_MAX_POS;
        baby_min_endstop[axis] = Z_BABY_DEFAULT_MIN_POS;
      }
      #ifdef BABYSTEP_XY
        else if (axis == X_AXIS) {
          baby_max_endstop[axis] = X_BABY_DEFAULT_MAX_POS;
          baby_min_endstop[axis] = X_BABY_DEFAULT_MIN_POS;
        }
        else if (axis == Y_AXIS) {
          baby_max_endstop[axis] = Y_BABY_DEFAULT_MAX_POS;
          baby_min_endstop[axis] = Y_BABY_DEFAULT_MIN_POS;
        }
      #endif //BABYSTEP_XY
    #endif //BABYSTEPPING
```
http://www.marlinfirmware.org/index.php/Coding_Standards